obelisk/cpp/zoo/CMakeLists.txt

```diff
@@ -22,7 +22,20 @@ add_library(Zoo INTERFACE)
 add_library(Obelisk::Zoo ALIAS Zoo)
 
 target_include_directories(Zoo INTERFACE include)
-target_include_directories(Zoo INTERFACE robot)
+
+# conditionally add leap subdirectory if OBELISK_BUILD_LEAP is set
```

Comment:
Maybe we should add a `message` line that says which options are on/off. I find that down the line this is useful to have so you can be sure certain flags are set and correct things are building.

Reply:
don't i have one right underneath each if statement already?
```
message(STATUS "Configuring Leap Hand Interface")
```
or are you saying we should always print even if the if statement is not true?

Reply:
resolved in person.